### PR TITLE
🌱 cmd/test-server/kcp/shard.go: use contextual logging

### PR DESCRIPTION
**Pull request by ncdc**
## Summary
cmd/test-server/kcp/shard.go: use contextual logging

## Related issue(s)

Part of #1694

### CODE REVIEW

File: cmd/test-server/kcp/shard.go [hunk 1]
```diff
@@ -138,10 +140,11 @@ func Start(ctx context.Context, name, runtimeDir, logFilePath string, args []str
 		}
 		time.Sleep(time.Millisecond * 1000)
 	}
-	klog.Infof("Found %s", kubeconfigPath)
+	logger.Info("Found kubeconfig", "path", kubeconfigPath)
 
+	logger = logger.WithValues("shard", name)
```
Comment (stevekuznetsov) [inline cmd/test-server/kcp/shard.go:145]: Can we add this as high up as possible?

**Review by stevekuznetsov — COMMENTED** [review]
LGTM modulo one nit

File: cmd/test-server/kcp/shard.go (hunk 1 above)
Comment (ncdc) [reply cmd/test-server/kcp/shard.go]: Moved up

**Comment by stevekuznetsov** [comment]
@ncdc just needs an update of the logcheck file

**Comment by ncdc** [comment]
@stevekuznetsov updated

**Comment by ncdc** [comment]
#1994 
/test test

**Pull request merged**